### ISSUE TITLE
python3.6: Incorrect installation layout

### ISSUE THREAD

**Issue by biojppm** [biojppm/cmany#4]
cmany expects the following layout:

```python
if util.in_windows():
    SHARE_DIR = osp.abspath(
        osp.join(osp.dirname(sys.executable), 'share', 'c4', 'cmany')
    )
else:
    SHARE_DIR = osp.abspath(
        osp.join(osp.dirname(osp.dirname(sys.executable)), 'share', 'c4', 'cmany')
    )
CONF_DIR = osp.join(SHARE_DIR, 'conf')
DOC_DIR = osp.join(SHARE_DIR, 'doc')
```
But a vanilla pip install (`pip 9.0.1 from /usr/lib/python3/dist-packages (python 3.6)`) creates the following:

global install:

```
  /usr/local/bin/cmany
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__init__.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/__init__.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/architecture.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/args.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/build.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/build_flags.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/build_item.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/build_type.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/cmake.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/combination_rules.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/compiler.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/conan.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/conf.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/flags.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/generator.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/help.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/main.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/named_item.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/project.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/system.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/util.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/variant.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/__pycache__/vsinfo.cpython-36.pyc
  /usr/local/lib/python3.6/dist-packages/c4/cmany/architecture.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/args.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/build.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/build_flags.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/build_item.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/build_type.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/cmake.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/combination_rules.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/compiler.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/conan.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/conf.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/flags.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/generator.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/help.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/main.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/named_item.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/project.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/system.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/util.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/variant.py
  /usr/local/lib/python3.6/dist-packages/c4/cmany/vsinfo.py
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2-py3.5-nspkg.pth
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/DESCRIPTION.rst
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/INSTALLER
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/METADATA
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/RECORD
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/WHEEL
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/entry_points.txt
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/metadata.json
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/namespace_packages.txt
  /usr/local/lib/python3.6/dist-packages/cmany-0.1.2.dist-info/top_level.txt
  /usr/local/share/c4/cmany/LICENSE.txt
  /usr/local/share/c4/cmany/README.rst
  /usr/local/share/c4/cmany/conf/cmany.yml
  /usr/local/share/c4/cmany/doc/build_items.txt
  /usr/local/share/c4/cmany/doc/cmany.txt
  /usr/local/share/c4/cmany/doc/dependencies.txt
  /usr/local/share/c4/cmany/doc/excluding_builds.txt
  /usr/local/share/c4/cmany/doc/flags.txt
  /usr/local/share/c4/cmany/doc/index.txt
  /usr/local/share/c4/cmany/doc/installing.txt
  /usr/local/share/c4/cmany/doc/quick_tour.txt
  /usr/local/share/c4/cmany/doc/reusing_arguments.txt
  /usr/local/share/c4/cmany/doc/vs.txt
  /usr/local/share/c4/cmany/requirements.txt
  /usr/local/share/c4/cmany/requirements_test.txt
```

local install:
```
  /home/jack/.local/bin/cmany
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__init__.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/__init__.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/architecture.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/args.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/build.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/build_flags.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/build_item.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/build_type.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/cmake.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/combination_rules.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/compiler.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/conan.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/conf.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/flags.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/generator.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/help.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/main.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/named_item.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/project.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/system.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/util.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/variant.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/__pycache__/vsinfo.cpython-36.pyc
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/architecture.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/args.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/build.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/build_flags.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/build_item.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/build_type.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/cmake.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/combination_rules.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/compiler.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/conan.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/conf.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/flags.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/generator.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/help.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/main.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/named_item.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/project.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/system.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/util.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/variant.py
  /home/jack/.local/lib/python3.6/site-packages/c4/cmany/vsinfo.py
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2-py3.5-nspkg.pth
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/DESCRIPTION.rst
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/INSTALLER
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/METADATA
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/RECORD
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/WHEEL
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/entry_points.txt
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/metadata.json
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/namespace_packages.txt
  /home/jack/.local/lib/python3.6/site-packages/cmany-0.1.2.dist-info/top_level.txt
  /home/jack/.local/share/c4/cmany/LICENSE.txt
  /home/jack/.local/share/c4/cmany/README.rst
  /home/jack/.local/share/c4/cmany/conf/cmany.yml
  /home/jack/.local/share/c4/cmany/doc/build_items.txt
  /home/jack/.local/share/c4/cmany/doc/cmany.txt
  /home/jack/.local/share/c4/cmany/doc/dependencies.txt
  /home/jack/.local/share/c4/cmany/doc/excluding_builds.txt
  /home/jack/.local/share/c4/cmany/doc/flags.txt
  /home/jack/.local/share/c4/cmany/doc/index.txt
  /home/jack/.local/share/c4/cmany/doc/installing.txt
  /home/jack/.local/share/c4/cmany/doc/quick_tour.txt
  /home/jack/.local/share/c4/cmany/doc/reusing_arguments.txt
  /home/jack/.local/share/c4/cmany/doc/vs.txt
  /home/jack/.local/share/c4/cmany/requirements.txt
  /home/jack/.local/share/c4/cmany/requirements_test.txt
```

**Comment by biojppm** [comment]
A working installation:
```
  c:\python36\lib\site-packages\c4\cmany\architecture.py
  c:\python36\lib\site-packages\c4\cmany\args.py
  c:\python36\lib\site-packages\c4\cmany\build.py
  c:\python36\lib\site-packages\c4\cmany\build_flags.py
  c:\python36\lib\site-packages\c4\cmany\build_item.py
  c:\python36\lib\site-packages\c4\cmany\build_type.py
  c:\python36\lib\site-packages\c4\cmany\cmake.py
  c:\python36\lib\site-packages\c4\cmany\combination_rules.py
  c:\python36\lib\site-packages\c4\cmany\compiler.py
  c:\python36\lib\site-packages\c4\cmany\conan.py
  c:\python36\lib\site-packages\c4\cmany\conf.py
  c:\python36\lib\site-packages\c4\cmany\flags.py
  c:\python36\lib\site-packages\c4\cmany\generator.py
  c:\python36\lib\site-packages\c4\cmany\help.py
  c:\python36\lib\site-packages\c4\cmany\main.py
  c:\python36\lib\site-packages\c4\cmany\named_item.py
  c:\python36\lib\site-packages\c4\cmany\project.py
  c:\python36\lib\site-packages\c4\cmany\system.py
  c:\python36\lib\site-packages\c4\cmany\util.py
  c:\python36\lib\site-packages\c4\cmany\variant.py
  c:\python36\lib\site-packages\c4\cmany\vsinfo.py
  c:\python36\lib\site-packages\cmany-0.1.2-py3.5-nspkg.pth
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\description.rst
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\entry_points.txt
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\installer
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\metadata
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\metadata.json
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\namespace_packages.txt
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\record
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\top_level.txt
  c:\python36\lib\site-packages\cmany-0.1.2.dist-info\wheel
  c:\python36\scripts\cmany.exe
  c:\python36\share\c4\cmany\conf\cmany.yml
  c:\python36\share\c4\cmany\doc\build_items.txt
  c:\python36\share\c4\cmany\doc\cmany.txt
  c:\python36\share\c4\cmany\doc\dependencies.txt
  c:\python36\share\c4\cmany\doc\excluding_builds.txt
  c:\python36\share\c4\cmany\doc\flags.txt
  c:\python36\share\c4\cmany\doc\index.txt
  c:\python36\share\c4\cmany\doc\installing.txt
  c:\python36\share\c4\cmany\doc\quick_tour.txt
  c:\python36\share\c4\cmany\doc\reusing_arguments.txt
  c:\python36\share\c4\cmany\doc\vs.txt
  c:\python36\share\c4\cmany\license.txt
  c:\python36\share\c4\cmany\readme.rst
  c:\python36\share\c4\cmany\requirements.txt
  c:\python36\share\c4\cmany\requirements_test.txt
```

**Comment by biojppm** [comment]
This was fixed in [cc55424](https://github.com/biojppm/cmany/commit/cc55424c4edd8a1368d5405c6d82089094544a82), published in [version 0.1.3](https://pypi.org/project/cmany/0.1.3/). The data files are now installed together with the source files.
